pkg/controllers/user/logging/utils/daemonset.go

```diff
@@ -151,11 +192,19 @@ func newFluentdDaemonset(name, namespace, clusterName string) *v1beta2.DaemonSet
 									Name:      "projectlogging",
 									MountPath: "/fluentd/etc/config/project",
 								},
+								{
+									Name:      "clustercustomlogconfig",
+									MountPath: "/fluentd/etc/config/customer/cluster",
```

Comment:
the `customer` in the path looks odd to me. Is it a typo for `custom`?

Reply:
ok